From 556cf184fc9a807e5d58a5306dd1339971f7fc14 Mon Sep 17 00:00:00 2001
Message-Id: <patch-2.2-556cf184fc9-20211021T183627Z-avarab@gmail.com>
In-Reply-To: <cover-0.2-00000000000-20211021T183627Z-avarab@gmail.com>
References: <cover-0.2-00000000000-20211021T183627Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 21 Oct 2021 13:24:56 +0200
Subject: [PATCH 2/2] git-sh-setup: remove messaging supporting
 --preserve-merges
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove messages that were last used by the code removed in
a74b35081c5 (rebase: drop support for `--preserve-merges`,
2021-09-07).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 git-sh-setup.sh | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/git-sh-setup.sh b/git-sh-setup.sh
index cee053cdc38..960982f9d53 100644
--- a/git-sh-setup.sh
+++ b/git-sh-setup.sh
@@ -217,9 +217,6 @@ require_clean_work_tree () {
 	then
 		action=$1
 		case "$action" in
-		rebase)
-			gettextln "Cannot rebase: You have unstaged changes." >&2
-			;;
 		"rewrite branches")
 			gettextln "Cannot rewrite branches: You have unstaged changes." >&2
 			;;
@@ -235,14 +232,7 @@ require_clean_work_tree () {
 		if test $err = 0
 		then
 			action=$1
-			case "$action" in
-			rebase)
-				gettextln "Cannot rebase: Your index contains uncommitted changes." >&2
-				;;
-			*)
-				eval_gettextln "Cannot \$action: Your index contains uncommitted changes." >&2
-				;;
-			esac
+			eval_gettextln "Cannot \$action: Your index contains uncommitted changes." >&2
 		else
 		    gettextln "Additionally, your index contains uncommitted changes." >&2
 		fi
-- 
2.33.1.1494.g88b39a443e1

